lib/book.rb

```diff
@@ -0,0 +1,18 @@
+class Book
+  attr_reader :title
+
+  def initialize(book_details)
+    @author_first_name = book_details[:author_first_name]
+    @author_last_name = book_details[:author_last_name]
+    @title = book_details[:title]
+    @publication_date = book_details[:publication_date]
+  end
+
+  def author
+    @author_first_name + " " + @author_last_name
+  end
+
+  def publication_year
+    @publication_date[-4..-1]
+  end
+end
```

Comment:
good job here, we *could* have done the work in your author/pub_year methods in the initialize method since we only need to do this work once per book